packages/frontend-2/pages/settings/workspaces/[slug]/members.vue

```diff
@@ -8,20 +8,20 @@
         class="mb-6"
       />
       <LayoutTabsHorizontal v-model:active-item="activeTab" :items="tabItems">
-        <template #default="{ activeItem }">
+        <template v-if="workspace" #default="{ activeItem }">
```

Comment:
I dislike the fact that you need to load to completion the workspace GQL query before you can even start mounting the subsequent components and their queries. This is why I think I'd rather rely on the apollo.cache hack

Reply:
With some refactoring I was able to just rely on just the slug, so we don't need the hack to get the ID